sentry_sdk/tracing.py

```diff
@@ -1432,49 +1433,16 @@ def timestamp(self):
         return convert_from_otel_timestamp(end_time)
 
     def start_child(self, **kwargs):
-        # type: (str, **Any) -> POTelSpan
+        # type: (**Any) -> POTelSpan
         kwargs.setdefault("sampled", self.sampled)
 
         span = POTelSpan(**kwargs)
         return span
 
-    @classmethod
-    def continue_from_environ(
```

Comment:
Please also add these 3 (`continue_from_x`, `from_traceparent`) to the migration guide. If we shim them we can remove them from the guide but I don't want us to forget adding them